<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Time Tracker v0.1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTimeEdit" name="initTimeEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>118</width>
      <height>26</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="displayFormat">
     <string>hh:mm</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="finalTimeEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>118</width>
      <height>26</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="displayFormat">
     <string>hh:mm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addTimeButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(143, 240, 164);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Añadir tarea</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="descriptionEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Descripción de la tarea</string>
    </property>
   </widget>
   <widget class="QTableView" name="taskTable">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>30</y>
      <width>452</width>
      <height>271</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>90</number>
    </attribute>
   </widget>
   <widget class="QPushButton" name="deleteTimeButton">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>110</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 117, 117);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Borrar tarea</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
